docs/items.txt

```diff
@@ -0,0 +1,11 @@
+Item format description:
+Space separated list of individual items ids
+Each id comprised of the following parts (parts surrounded with [] are optional, parts surrounded in <> are to be replaced as per table defined at the end of this document):
+(pay attention to the special chars like | or *)
+[<chance>|][<amount>*]<itemName>[-<slot>][+<itemName>[-<slot>]][+<itemName>[-<slot>]][+<itemName>[-<slot>]]
+The end may optionally continue even with other itemnames.
+chance is a float between 0 and 1, denoting how probable it is ship will receive the item. 0 means not at all probably, 1 means completely sure. Defaults to 1 when not specified
+amount is integer denoting how much of the item player will receive, should the chance check pass. must be >=1, defaults to 1 when not specified
+itemName is the gestalt id of the item, in standard form "moduleName:itemName"
```

Comment:
Maybe there should be a temporary note here about the existence of some non-gestalt ids? (The repair item `rep` being the primary example)

Reply:
With code modules, better course of action might actually be gestaltifying `rep` and moving it to core.

Reply:
Okay, in that case after this is merged we should make a separate issue for that. 👍